tcib/builder/buildah.py

```diff
@@ -179,6 +179,7 @@ def build(self, container_name, container_build_path):
         if self.debug:
             # TODO(bogdando): add --log-rusage for newer buildah
             bud_args.extend(['--loglevel=3'])
+        bud_args.extend(['--label', 'tcib_build_tag=%s' % self.tag])
```

Comment:
Thank you for the pull request. Based on my understanding and 
 [code search ](https://github.com/search?q=repo%3Aopenstack-k8s-operators%2Ftcib%20label&type=code), It is already implemented in TCIB.

openstack tcib container image build ----label does the same.

Sorry I am not getting from this pr, can we add an example for the same.
TCIB also provides support for setting tags using --tag flag.

Reply:
Thank you steve for explaining it detail. Make sense now.